docs/_includes/lhn-template.html

```diff
@@ -13,7 +13,7 @@
         {% if hub.href == activeHub %}
             <li>
                 <div class="icon-with-link selected">
-                    <i class="fa-solid fa-angle-down icon"></i>
+                    <img src="../assets/images/down.svg" class="icon icon-down"></img>
                     <span>{{ hub.title }}</span>
                 </div>
                 <ul class="nested-treeview">
```

Comment:
Weird linking compared to other linking in default.html

Reply:
@Santhosh-Sellavel  I just fixed it please check again